internal/appsec/dyngo/instrumentation/httpinstr/tags.go

```diff
@@ -0,0 +1,91 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package httpinstr
+
+import (
+	"encoding/json"
+	"sort"
+	"strings"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
+)
+
+// setEventSpanTags sets the security event span tags into the service entry span.
+func setEventSpanTags(span ddtrace.Span, events json.RawMessage) {
+	// Set the appsec event span tag
+	// eventTag is the structure to use in the `_dd.appsec.json` span tag.
+	type eventTag struct {
+		Triggers json.RawMessage `json:"triggers"`
+	}
+	// TODO(Julio-Guerra): avoid serializing the json in the request hot path
+	event, err := json.Marshal(eventTag{Triggers: events})
```

Comment:
Perhaps it's not relevant here, but if you think there may be situations where this may be called many times over a short period of time, you may want to literally write out the JSON here instead of calling `json.Marshal`. It gives a big performance boost.

Reply:
I'm interested by any low hanging fruit here ^^
Do you mean to manually create the json string myself with for something like `strings.Builder`?